moveit_core/planning_scene/test/test_multi_threaded.cpp

```diff
@@ -44,6 +44,8 @@
 #include <moveit/collision_detection/collision_common.h>
 #include <moveit/collision_detection/collision_env.h>
 #include <moveit/collision_detection/collision_detector_allocator.h>
+#include <moveit/collision_detection_bullet/collision_detector_allocator_bullet.h>
+#include <moveit/collision_detection_bullet/collision_env_bullet.h>
```

Comment:
These includes are not available when Bullet is not available. You need to conditionally include them!
My suggestion is to add add_definitions(-DBULLET_ENABLE) [here](https://github.com/ros-planning/moveit/blob/4d6f805b8375104fd5dac4791032a9c11353fffe/moveit_core/CMakeLists.txt#L36-L40) and in c++ code use:
```cpp
#ifdef BULLET_ENABLE
...
#endif
```

Reply:
Thank you for the suggestion! Added in 2433753